ESP32LapTimer/src/targets/config_default.h

```diff
@@ -5,12 +5,7 @@
 #define BUTTON1 T7 // 27
 #define BUTTON2 T4 // 13
 
-#define CS1 16
-#define CS2 5
-#define CS3 4
-#define CS4 15
-#define CS5 25
-#define CS6 26
+#define CS_PINS 16, 5, 4, 15, 25, 26
```

Comment:
The CS pins and ADC channels currently have a 1:1 relationship. Any plan on how to solve this since we only have 6 ADC channels?

Reply:
In a hypothetical future where i manage to get some kind of mux solution working where one can use up to 8 receivers i imagine the ADC stuff being behind a feature flag or something along those lines but for now i dont think anything needs to change.
As far as i can tell the ADC definitions are not dependent on the CS ones right now.

Reply:
They are not syntactically but semantically dependend. The RSSI from module 1 is always read from ADC1.

Reply:
I think a note explaining that ordering is important would suffice since that’s how it gets allocated